notebooks/MAST/plot_analyze_with_lightkurve/plot_analyze_with_lightkurve.ipynb

```diff
@@ -0,0 +1,953 @@
+{
```

Comment:
<p>Here (and in Part 1), may be useful to refer to the star by its TIC ID in addition to giving the coordinates, e.g., "TIC 373283207 (RA = 2.238638, Dec = 73.690475)"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/13/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
<p>I may just be having a silly moment, but what is TIC ID? For now I put the APOGEE ID from the catalog, explained as "TMASS-STYLE object name (2MHHMMSSss+DDMMSSs) (APOGEE_ID)"</p>

Reply:
<p>Oh - I found it farther down in the tutorial ;-) It's on the TESS spectra. However, at this point in the tutorial, the user knows there are TESS spectra but hasn't yet accessed them. Or was there an earlier place where this ID number would have been known? </p>

Reply:
<p>APOGEE ID is good too! Especially since the TESS *light curves haven't been accessed at this point, and it might not be clear what the TIC ID is yet.</p><p> </p><p>The TIC ID <em>is</em> given in Part 1 of the notebook ("target_name" in "masttbl" in the final cell) and is likely also given by one of the columns in your "obs_tbl" in the next cell. Up to you if you want to reference it here; I think APOGEE ID is perfect.</p>